crime-applications-adaptor/src/main/resources/schemas/criminalapplicationsdatastore/application.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
I don't think we are using this json , please verify

Reply:
same as other comment, see below :)

Reply:
done